### ISSUE TITLE
can't be render use relay

### ISSUE THREAD

**Issue by owcc** [styled-components/styled-components#1466]
```styled-components``` is a great practice for write css-in-js and also popular,  unfortunately, it's seems that there is little problem when use it with relay

```js
// home.js
...
export const Home = styled.div`
  background-color: rgba(0,0,0,.04);
`
```

``` js
// app.js
import { Home } from 'path/to/home'
...

class App extends React.Component {
  render(){
    return <QueryRenderer
    environment={AppEnvironment}
    query={appQuery}
    variables={{}}
    render={({ error, props }) => {
      if (error) {
        return <div>Error!</div>;
      }
      if (!props) {
        return <div>Loading...</div>;
      }
      return <Home>Home page</Home>
      // this will be work
      // return <div>Home page</div>
    }}
  />
  }
}
ReactDOM.render(createElement(App), document.getElementById('app'))
```

the page is render nothing and throw this warnning 
```
Warning: Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.
    in templateFunction (at index.jsx:40)
    in ReactRelayQueryRenderer (at index.jsx:28)
    in App
```

Any suggestions on this?

**Comment by probablyup** [comment]
Hmm, nothing is leaping out at me based on your example code. Would you happen to have a sample repository with the problem we could test?

**Comment by owcc** [comment]
thank you so much for reply, there is a very simple test to it, try this [example](https://codesandbox.io/s/z3l45v2wxp)

**Comment by k15a** [comment]
If you remove the additional function after export default it will work. So your code should look like `export default styled.div`.

**Comment by k15a** [comment]
Here is a working example: https://codesandbox.io/s/x3143rqpjp
